rules/external_binary/BUILD.bazel

```diff
@@ -1 +1,17 @@
 package(default_visibility = ["//visibility:public"])
+
+load(":def.bzl", "binary_location")
+
+[binary_location(
+    name = "{}_binary_location".format(name).replace("-", "_"),
```

Comment:
Would it make sense to just use the name as-is (without the `_binary_location` suffix)?

Reply:
Maybe. I'll try it.